src/onnxruntime.cc

```diff
@@ -596,6 +607,8 @@ class ModelInstanceState : public BackendModelInstance {
   // instance.
   OrtSession* session_;
   OrtAllocator* allocator_;
+  OrtIoBinding* io_binding_;
+  bool is_io_binding_enabled_;
```

Comment:
Is there any disadvantage to running with iobinding *always* enabled?

Reply:
changed this... switched to using io_binding aways